apollo-router/src/plugins/snapshots/apollo_router__plugins__expose_query_plan__tests__it_expose_query_plan-2.snap

```diff
@@ -127,36 +127,36 @@ expression: "serde_json::to_value(response).unwrap()"
                     "@",
                     "reviews",
                     "@",
-                    "product"
+                    "author"
                   ],
                   "node": {
                     "kind": "Fetch",
-                    "serviceName": "products",
+                    "serviceName": "accounts",
                     "requires": [
                       {
                         "kind": "InlineFragment",
-                        "typeCondition": "Product",
+                        "typeCondition": "User",
                         "selections": [
                           {
                             "kind": "Field",
                             "name": "__typename"
                           },
                           {
                             "kind": "Field",
-                            "name": "upc"
+                            "name": "id"
                           }
                         ]
                       }
                     ],
                     "variableUsages": [],
-                    "operation": "query TopProducts__products__2($representations:[_Any!]!){_entities(representations:$representations){...on Product{name}}}",
-                    "operationName": "TopProducts__products__2",
+                    "operation": "query TopProducts__accounts__2($representations: [_Any!]!) { _entities(representations: $representations) { ... on User { name } } }",
```

Comment:
what is happening here? Different order for the list of parallel fetch nodes?

Reply:
yep, and that’s expected. the rust qp outputs parallel fetches in a different order, and that’s semantically insignificant.